config/locales/devise_invitable.fr.yml

```diff
@@ -21,11 +21,12 @@ fr:
       updated_not_active: Votre mot de passe a été défini avec succès.
     mailer:
```

Comment:
I don't speak French and I've used Google Translate for this part. I'm not sure how accurate google translate is but since it was short sentences I thought it may be worth a try. 

If we feel that we would prefer to just keep it in English for now until we can human translate it, I can update this.

Reply:
Ella verified the text so I'm resolving this conversation.